panoptes_client/caesar.py

```diff
@@ -0,0 +1,126 @@
+from panoptes_client.panoptes import Panoptes
+
+
+class Caesar(object):
+    EXTRACTOR_TYPES = ['blank', 'external', 'question', 'survey', 'who', 'pluck_field', 'shape']
+    REDUCER_TYPES = ['consensus', 'count', 'placeholder', 'external', 'first_extract', 'stats', 'unique_count', 'rectangle', 'sqs']
+    RULE_TO_ACTION_TYPES = {
+        'subject': ['retire_subject', 'add_subject_to_set', 'add_to_collection', 'external'],
+        'user': ['promote_user']
+    }
+
+    def __init__(self, endpoint='https://caesar-staging.zooniverse.org', redirect_url='https://caesar.zooniverse.org/auth/zooniverse/callback'):
```

Comment:
This might be something you're planning to change before merging, but this should default to the production endpoint rather than staging.